nexus-sdk-js/src/File/index.ts

```diff
@@ -0,0 +1,45 @@
+import { FileResponse } from './types';
+import { createFile } from './utils';
+import { Context } from '../Resource/types';
+
+export default class File {
+  readonly orgLabel: string;
+  readonly projectLabel: string;
+  readonly context: Context;
+  readonly type: 'File';
+  readonly self: string;
+  readonly id: string;
+  readonly constrainedBy: string;
+  readonly project: string;
+  readonly createdAt: string;
+  readonly createdBy: string;
+  readonly updatedAt: string;
+  readonly updatedBy: string;
+  readonly rev: number;
+  readonly deprecated: boolean;
+  readonly raw: FileResponse;
+
+  static create = createFile;
```

Comment:
I would probably call this `File.upload` instead

Reply:
API docs say "Create a file", so I'm OK with reusing the same terminology

Reply:
doesn't match the docs buddy